pkg/kdump/kdump-view.jsx

```diff
@@ -414,13 +414,27 @@ export class KdumpPage extends React.Component {
 # A reboot will be required if crashkernel was not set before
 kdumpctl reset-crashkernel`;
         }
-        const shell = `
```

Comment:
Please prefix the "add instructions to enable kdump on microos systems" commit subject with "kdump:". And perhaps s/instructions/shell commands/ ?